src/executables/master_request.c

```diff
@@ -606,6 +606,51 @@ css_process_kill_immediate (CSS_CONN_ENTRY * conn, unsigned short request_id, ch
     }
 }
 
+/*
+ * css_process_start_shutdown_by_name()
+ *   return: none
+ *   server_name(in/out)
+ */
+void
+css_process_start_shutdown_by_name (char *server_name)
+{
+#if !defined(WINDOWS)
+  SOCKET_QUEUE_ENTRY *temp;
+  char buffer[512];
+  int rv = pthread_mutex_lock (&css_Master_socket_anchor_lock);
+
```

Comment:
warning 제거 목적이라면, (void) 추가 + 이 의도라면 pthread_mutex_unlock() 역시 동일할 처리 필요
```suggestion

  pthread_mutex_lock (&css_Master_socket_anchor_lock);
```

Reply:
수정하였습니다.